norse/torch/functional/adjoint/lif_refrac_adjoint.py

```diff
@@ -112,6 +113,122 @@ def lif_refrac_adjoint_step(
     return z, LIFRefracState(LIFState(z, v, i), rho)
 
 
+class LIFSparseAdjointRefractFunction(torch.autograd.Function):
```

Comment:
I would recommend choosing a consistent abbreviation for 'refractory'. Below, `Refrac` is used but here it's `Refract`.

Reply:
I have added some tests. Which test should be added yet

Reply:
Thanks for the changes. I was thinking about tests for the functional parts, but that's already been done I see!